packages/testing-toolkit/src/index.ts

```diff
@@ -0,0 +1,209 @@
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+
+export async function inputText(
+  canvasElement: HTMLElement,
+  linkId: string,
+  text: string | boolean | number
+) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+  const input =
+    questionElement?.querySelector('input') ?? questionElement?.querySelector('textarea');
+
+  if (!input) {
+    throw new Error(`Input or textarea was not found inside ${`[data-linkid=${linkId}] block`}`);
+  }
+
+  fireEvent.change(input, { target: { value: text } });
+
+  // Here we await for debounced store update
+  await new Promise((resolve) => setTimeout(resolve, 500));
+}
+export async function chooseCheckBox(canvasElement: HTMLElement, linkId: string) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+  const input =
+    questionElement?.querySelector('input') ?? questionElement?.querySelector('textarea');
+
+  if (!input) {
+    throw new Error(`Input or textarea was not found inside ${`[data-linkid=${linkId}] block`}`);
+  }
+
+  fireEvent.click(input);
+
+  // Here we await for debounced store update
+  await new Promise((resolve) => setTimeout(resolve, 500));
+}
+
+export async function inputFile(
+  canvasElement: HTMLElement,
+  linkId: string,
+  files: File | File[],
+  url: string,
+  filename: string
+) {
+  const questionElement = await findByLinkId(canvasElement, linkId);
+  const input = questionElement?.querySelector<HTMLInputElement>('input');
+  const textarea = questionElement?.querySelectorAll<HTMLInputElement>('textarea');
+
+  if (!input) {
+    throw new Error(`File input was not found inside [data-linkid=${linkId}] block`);
+  }
+  const fileList = Array.isArray(files) ? files : [files];
+  await userEvent.upload(input, fileList);
+
+  fireEvent.change(textarea[0], { target: { value: url } });
```

Comment:
add data-test props to inputs in Attachment item type instead of using 0, 2 magic numbers